sdk/sdk-api/src/main/java/mil/emp3/api/WMS.java

```diff
@@ -154,4 +156,20 @@ public void setLayers(List<String> newLayers) throws EMP_Exception {
     public List<String> getLayers() {
         return this.oLayers;
     }
+
+    /**
+     * Prints all configuration parameters
+     */
+    @Override
+    public String toString() {
+        String str = "URL: " + serviceURL + "\n"
+                + "WMS Version: " + eVersion.toString() + "\n"
+                + "Transparent: " + bTransparent + "\n"
+                + "Tile format: " + sTileFormat + "\n"
+                + "Layers: \n";
+        for (String layer : oLayers) {
+            str += "\t" + layer + "\n";
```

Comment:
Check thisgs like oLayers for null so that we don't get an exception because of "toString"